nomad/grapl-core.nomad

```diff
@@ -830,16 +858,18 @@ job "grapl-core" {
       driver = "docker"
 
       config {
-        image        = "${var.container_registry}grapl/analyzer-executor:${var.analyzer_executor_tag}"
-        network_mode = "grapl-network"
+        image = "${var.container_registry}grapl/${var.container_repo}analyzer-executor:${var.analyzer_executor_tag}"
+      }
+
+      template {
+        data        = local.local_vars
+        destination = "analyzer-executor.env"
+        env         = true
       }
 
       env {
         # AWS vars
-        AWS_REGION                  = var.aws_region
-        GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
-        GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
-        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
+        AWS_REGION = var.aws_region
```

Comment:
Since this is a Python app, this will _have_ to be `AWS_DEFAULT_REGION`; it probably only works because we've only ever run in `us-east-1`.

We should dig into https://github.com/grapl-security/issue-tracker/issues/301 at some point 😅